infra/prod/_modules/storage_account/storage_account_fims.tf

```diff
@@ -0,0 +1,16 @@
+module "storage_account_fims" {
+  source = "github.com/pagopa/terraform-azurerm-v3//storage_account?ref=v7.76.0"
+
+  name                          = "${var.prefix}${var.env_short}${var.domain}st"
```

Comment:
you can use directly `var.product` replacing `"-"` with `""`

Reply:
[ad0aa60](https://github.com/pagopa/io-fims/pull/24/commits/ad0aa60a06ab7bf42564f361291562e5db7d4d83)